mkosi

```diff
@@ -1785,6 +1785,13 @@ def install_debian_or_ubuntu(args: CommandLineArguments,
                "--exclude=sysv-rc,initscripts,startpar,lsb-base,insserv",
                "--components=" + ','.join(repos)]
 
+    # Atleast ubuntu doesn't have apt-transport-https anymore, except on xenial,
+    # but debootstrap scripting tries to install it.
+    # Not checking end-of-life releases
+    # Correct patch should be in debootstrap scripting
+    if args.distribution == Distribution.ubuntu and args.release not in ["xenial"]:
+        cmdline[4] += ",apt-transport-https"
+
```

Comment:
Sounds like a fix to debootstrap. Have you created a bug report?

Reply:
There has been bug report open for debootstrap since 2017 [debian bug 879755].
Now that I rechecked the status, there finally seems to be fix in debootstrap upstream for this since 2020-03-07 but this kind of workaround probably will be needed for LTS distributions for a while like current Ubuntu LTS and possibly next one, have to check if it's fixed there also and comment on the bug report already made for Ubuntu bug tracking too.